sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -107,8 +107,8 @@ export class ClientContext {
         "AzureKeyVault",
         this.encryptionKeyTimeToLiveInHours,
       );
-      this.encryptionSettingsCache = new EncryptionSettingsCache();
-      this.clientEncryptionKeyPropertiesCache = new ClientEncryptionKeyPropertiesCache();
+      this.encryptionSettingsCache = EncryptionSettingsCache.getInstance();
```

Comment:
We have made `encryptionSettingsCache` and `clientEncryptionPropertiesCache` as singleton. No need to store them in client context now.